src/main/resources/db/migration/V1_0__Create_User.sql

```diff
@@ -0,0 +1,11 @@
+--------------
+---- User ----
+--------------
+
+create table if not exists users (
+    id          serial constraint user_pk primary key,
+    first_name   varchar(50) not null,
+    last_name    varchar(50) not null,
+    email       varchar(50) not null constraint email_unique unique,
```

Comment:
add index on email and last_name as we will later search users by these keys

Explanation here https://www.facebook.com/409699499865702/videos/565820090983104